hw/lint/data/veriblelint.hjson

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    tool_srcs: []
+
+    // TODO(#1342): pass Verible config file to FuseSoC, once supported by Verible
+    // We rely on fusesoc to run lint for us
+    build_cmd:  "fusesoc"
+    build_opts: ["--cores-root {proj_root}",
+                 "run",
+                 "--target={flow}",
+                 "--tool={tool}",
+                 "--build-root={build_dir}",
+                 "{fusesoc_core}"]
+
+    // TODO(#1342): switch over to native structured tool output, once supported by Verible
+    // Verible lint-specific results parsing script that is called after running lint
+    report_cmd: "{proj_root}/hw/lint/tools/{tool}/parse-lint-report.py "
+    report_opts: ["--repdir={build_dir}",
+                  "--outdir={build_dir}"]
```

Comment:
This seems exactly identical to ascentlint cfg.

Reply:
This actually was located in the common lint config file initially, but I have split this out since the report command options are different for Verible, and we plan to update both the report command and the build command in the near future (mind the TODO's ;)). 

But thinking about this some more, I think we can leave the FuseSoC build command aligned, since the Verible configuration file will likely be defined in a common FuseSoC lint core file (similar to how this has been done for AscentLint). 

Will adjust that.

Reply:
Okay, sounds good.